.gitignore

```diff
@@ -1,2 +1,4 @@
 .DS_Store
-workshop/themes/*
\ No newline at end of file
+workshop/themes/*
```

Comment:
We can't have the themes checked-in into GitHub because is third-party software. Can you remove this line pls `workshop/themes/*`?

Reply:
This is the .gitignore so its make sure that those files aren't checked in.
I added a couple more based the ide I'm using.